cypress/tests/integration/stakeholder/[stakeholderid].cy.js

```diff
@@ -0,0 +1,34 @@
+import stakeholder from '../../../../doc/examples/organizations/1.json';
+import { prepareNocks, clearNocks } from '../nockRoutes';
+
+beforeEach(() => {
+  clearNocks();
+});
+
+describe('Organizations', () => {
+  const imageFixturePath = `images/organization.png`;
+  return it(`organization test`, () => {
+    const path = `/stakeholder/${stakeholder.id}`;
+    cy.fixture(imageFixturePath).then((image) => {
+      const blob = Cypress.Blob.base64StringToBlob(image, 'images/png');
+      const photo_url = Cypress.Blob.createObjectURL(blob);
+      prepareNocks({ stakeholder: { ...stakeholder, photo_url } });
+    });
+
+    cy.task('nockIntercept', {
+      hostname: 'https://dev-k8s.treetracker.org',
+      method: 'get',
+      path: '/query/v2/stakeholder/1',
+      statusCode: 200,
+      body: stakeholder,
```

Comment:
The organization mock json is deprecated, please create a new fixture (under fixture folder) json file for stakeholder , the spec of stakeholder is defined in our query api doc here:
https://github.com/Greenstand/treetracker-query-api/blob/fa194cb4a274ee688c2b26b58596147368d08b9e/docs/api/spec/query-api.yaml#L1542-L1559